### ISSUE TITLE
What does sr_nm stand for?

### ISSUE THREAD

**Issue by Zhang-kg** [UDC-GAC/venom#1]
What does sr_nm stand for? 
the VNM sparsity format?

**Comment by LopezCastroRoberto** [comment]
Yes, it was an initial name candidate for VNM. Thank you for pointing out the typo.

**Comment by Zhang-kg** [comment]
您好，您的邮件我已经收到！

**Comment by Zhang-kg** [comment]
I am wondering the relationship between format_sr_nm and blockwise_format_origin/blockwise_format, could you please explain? thx

**Comment by Zhang-kg** [comment]
And I am also wondering the init function and the exec function. These two functions are not explained in the paper. Could you please provide a specific explanation of their functions?
